app/src/main/res/drawable/ic_location_marker.xml

```diff
@@ -4,12 +4,12 @@
     android:viewportWidth="512.01"
     android:viewportHeight="512.01">
   <path
-      android:fillColor="#FF000000"
+      android:fillColor="#285294"
       android:pathData="M388.89,313.055c-11.435,-2.773 -22.955,4.373 -25.664,15.829c-2.731,11.456 4.373,22.955 15.829,25.664c66.261,15.723 90.283,38.976 90.283,50.795c0,25.493 -85.077,64 -213.333,64c-128.235,0 -213.333,-38.507 -213.333,-64c0,-11.819 24.043,-35.072 90.261,-50.795c11.477,-2.709 18.56,-14.208 15.829,-25.664c-2.709,-11.456 -14.229,-18.603 -25.664,-15.829C43.717,331.871 0.005,364.639 0.005,405.343c0,70.016 128.811,106.667 256,106.667c127.211,0 256,-36.651 256,-106.667C512.005,364.639 468.293,331.871 388.89,313.055z"/>
   <path
-      android:fillColor="#FF000000"
+      android:fillColor="#285294"
       android:pathData="M256.015,170.681c11.776,0 21.333,-9.557 21.333,-21.333s-9.557,-21.333 -21.333,-21.333s-21.333,9.557 -21.333,21.333S244.239,170.681 256.015,170.681z"/>
   <path
-      android:fillColor="#FF000000"
+      android:fillColor="#285294"
       android:pathData="M228.239,397.518l8.683,17.365c3.627,7.232 11.008,11.797 19.093,11.797s15.467,-4.565 19.093,-11.797l18.389,-36.779c22.379,-44.779 49.984,-88.149 76.672,-130.091l12.16,-19.136c15.061,-23.808 23.019,-51.307 23.019,-79.531c0,-42.496 -18.197,-83.115 -49.92,-111.445C323.727,9.592 281.082,-3.975 238.565,1.017C172.175,8.569 117.349,62.03 108.154,128.121c-5.44,39.168 4.352,78.059 27.541,109.547C171.877,286.691 200.122,341.241 228.239,397.518zM256.015,85.347c35.285,0 64,28.715 64,64c0,35.285 -28.715,64 -64,64s-64,-28.715 -64,-64C192.015,114.062 220.73,85.347 256.015,85.347z"/>
```

Comment:
These changes should be reverted.

![image](https://user-images.githubusercontent.com/23643192/113224913-75713f00-925a-11eb-8b38-efeabbab0d2a.png)

The location marker isn't a button, so it shouldn't be blue. It also looks weird in my opinion.
What should be blue are the buttons that contain the texts. You can't see it very well, but the texts such as "Select Group", and "Select Location" are buttons. Those are the UI components that should be blue.

@Landry333 What do you think?